### PR TITLE
#1753 - Add student notes during add manual overaward deduction - part 1

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- DB changes for the ticket #1753:
  - added note_id, added_by and added_date to sims. disbursement_overawards table.

### CODE REVIEW

File: sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- Add column note_id to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS note_id INTEGER REFERENCES sims.notes(id);
```
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql:5]: I missed the same on the previous PR, but most of the time I believe that we use the types like `INT`.

**Review by andrewsignori-aot — COMMENTED** [review]
Overall looks good, just one minor comment 👍 
Just as a suggestion, while doing migration specific I would like to have the changes on the entity models classes altogether also to be reviewd, when possible.

**Review by ann-aot — APPROVED** [review]
👍 Good work @andrepestana-aot

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ADD
+    COLUMN IF NOT EXISTS added_by INT REFERENCES sims.users(id);
+
+COMMENT ON COLUMN sims.disbursement_overawards.added_by IS 'User id of the user adding a manual record.';
+
+-- Add column added_date to sims.disbursement_overawards.
+ALTER TABLE
+    sims.disbursement_overawards
+ADD
+    COLUMN IF NOT EXISTS added_date TIMESTAMP WITHOUT TIME ZONE;
```
Comment (dheepak-aot) [inline sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql:21]: Why is the timestamp without time zone? 

All our assessed_date follow timestamp with time zone.

image

**Comment by dheepak-aot** [comment]
Nice work. Just one comment. 👍

File: sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql (hunk 2 above)
Comment (andrepestana-aot) [reply sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Add-note-id-added-by-added-date.sql:21]: OK. You're right. Thanks!

**Review by dheepak-aot — APPROVED** [review]
Thanks for making the change. LGTM 👍

**Pull request merged**